float64 delta_rot1
float64 delta_rot2
float64 delta_trans
float64[9] covariance
